CMakeLists.txt

```diff
@@ -24,13 +24,17 @@ else()
   set(SCRIPT_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/cmake")
 endif()
 
-configure_file(cmake/PybindWrap.cmake.in
-               ${CMAKE_CURRENT_SOURCE_DIR}/cmake/PybindWrap.cmake @ONLY)
 # Install scripts
 install(FILES cmake/PybindWrap.cmake cmake/gtwrapConfig.cmake
               cmake/GtwrapUtils.cmake
         DESTINATION "${SCRIPT_INSTALL_DIR}/gtwrap")
 
+# Install wrapping scripts
+install(PROGRAMS scripts/pybind_wrap.py scripts/matlab_wrap.py TYPE BIN)
+
+# Install pybind11 directory
+install(DIRECTORY pybind11 TYPE LIB)
```

Comment:
So, the final destination will be `<ROOT>/lib/gtwrap/pybind11`? Please list a final changeset when we run `make install`, where will the files be installed?

Reply:
*ideally* here should be a comment describing this.

Reply:
ROOT will be whatever CMAKE_INSTALL_PREFIX is. And wouldn't the install_manifest.txt have that entire changeset?

Reply:
I know, I mean this would be friendly to new users (single point of information)